app/components/combined_system_message/combined_system_message.js

```diff
@@ -304,38 +304,51 @@ export default class CombinedSystemMessage extends React.PureComponent {
         );
     }
 
+    renderMessage(postType, userIds, actorId, style) {
+        return (
+            <React.Fragment key={postType + actorId}>
```

Comment:
Is the React.Fragment needed here?

Reply:
Yes, needed since not using any other container.  Without it, React complains requiring that to have the key.

Reply:
I think the key could be added in renderFormattedMessage then, but this is fine as well